packages/dds/tree/src/test/shared-tree/fuzz/topLevel.fuzz.spec.ts

```diff
@@ -87,7 +87,7 @@ describe("Fuzz - Top-Level", () => {
 				directory: failureDirectory,
 			},
 			clientJoinOptions: {
-				clientAddProbability: 0,
+				clientAddProbability: 1,
```

Comment:
this is a probability of generating a client add per operation (roughly speaking, the doc comment will have more details), so you probably want something like `0.05` to `0.1` here.